pkg/odo/cli/storage/create.go

```diff
@@ -55,9 +57,17 @@ func (o *StorageCreateOptions) Validate() (err error) {
 
 // Run contains the logic for the odo storage create command
 func (o *StorageCreateOptions) Run() (err error) {
-	_, err = storage.Create(o.Client, o.storageName, o.storageSize, o.storagePath, o.Component(), o.Application)
+	storageResult, err := storage.Create(o.Client, o.storageName, o.storageSize, o.storagePath, o.Component(), o.Application)
 	if err == nil {
```

Comment:
what is there was an error? No error output?

Reply:
It will call and return the error to `util.LogErrorAndExit(o.Run(), "")` in GenericRun() in runnable.go

So it will log the error and and exit